modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -515,3 +515,52 @@
 	glass_desc = "Looking at it makes you recall every mistake you’ve made."
 	quality = DRINK_VERYGOOD //takes extra effort
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/shakiri
+	name = "Shakiri"
+	description = "A sweet, fragrant red drink made from fermented kiri fruits. It seems to gently sparkle when exposed to light."
+	boozepwr = 45
+	color = "##cf3c3c"
```

Comment:
```suggestion
	color = "#cf3c3c"
```
This is why shakiri can't mix successfully.

Reply:
oops! my bad! thanks for catching that, couldn't figure out why it was failing haha